Source/Project64/UserInterface/SettingsConfig.cpp

```diff
@@ -251,9 +251,9 @@ void CSettingConfig::ApplySettings(bool UpdateScreen)
     if (!GameIni.empty())
     {
         stdstr GoodName;
-        if (!g_Settings->LoadStringVal(Game_GoodName, GoodName))
+        if (!g_Settings->LoadStringVal(Cfg_GoodName, GoodName))
```

Comment:
should be 

if (g_Settings->LoadStringVal(Rdb_GoodName, GoodName))

Since Cfg_GoodName does not inherit from Rdb_GoodName it would be always writing an empty string to the cfg file